src/sentry/notifications/activity/base.py

```diff
@@ -177,9 +173,9 @@ def _get_user_avatar_url(self, user, size=20):
         url = reverse("sentry-user-avatar-url", args=[avatar.ident])
         if size:
             url = f"{url}?s={int(size)}"
-        return absolute_uri(url)
+        return str(absolute_uri(url))
```

Comment:
same here, why is this a string now? is it just because of the typing?

Reply:
Yeah it's for types. I marked `absolute_uri` as returning an `str` but `mypy` doesn't believe me.